.github/workflows/check_python_methods.py

```diff
@@ -111,11 +111,11 @@ def parse(type, names):
                     description_extras[field] = extras_values
 
                 # Parameters
-                method_text.append("**Parameters:** \n")
+                method_text.append("**Parameters:**\n\n")
                 no_parameters = True
                 if "Parameters:" in description_extras:
                     no_parameters = False
-                    extras_values = description_extras["Parameters:"]
+                    extras_values = description_extras["**Parameters:**\n\n"]
```

Comment:
@npentrel Sorry I couldn't suggest so I figured it'd be ok to commit directly in this case, added spacing here too because it was formatting some just blank

Reply:
Actually that might not be correct let me check

Reply:
Sorry, this did not fix the problem -- reverting